pyphetools/creation/hpo_parser.py

```diff
@@ -6,7 +6,7 @@
 import os
 
 
-HPO_JSON_URL = "https://raw.githubusercontent.com/obophenotype/human-phenotype-ontology/master/hp.json"
+HPO_JSON_URL = "https://raw.githubusercontent.com/obophenotype/human-phenotype-ontology/f99e141f67d79e2521d0f5782be9b2559572128c/hp.json"
```

Comment:
```suggestion
HPO_JSON_URL = "http://purl.obolibrary.org/obo/hp/hp.json"
```

Reply:
We should never use URLs in code - always use PURLs (_persistent_ URLs). These are guaranteed to always work and redirect to the correct ontology file.